internal/model/live_update.go

```diff
@@ -20,23 +20,32 @@ func NewLiveUpdate(steps []LiveUpdateStep, fullRebuildTriggers PathSet) (LiveUpd
 		switch step.(type) {
 		case LiveUpdateSyncStep:
 			if seenRunStep {
-				return LiveUpdate{}, errors.New("all sync steps must precede all run steps")
+				return LiveUpdate{}, errors.New("live_update: all sync steps must precede all run steps")
```

Comment:
it seems a bit cleaner to leave the `live_update` off here and attach it in the tiltfile instead, so that function names are all in that package, but nbd